Assets/MixedRealityToolkit/Providers/UnityInput/MouseController.cs

```diff
@@ -130,6 +126,16 @@ public void Update()
                         }
                     }
                 }
+
+                if (Interactions[i].InputType == DeviceInputType.Scroll)
```

Comment:
What's the benefit of moving this block?

Reply:
no benefit. just happened while merging the old with the new commits. if that's an issue I'll move it back? let me know

Reply:
Just easier to review without as much churn. No worries.